hibernate-reactive-core/src/test/java/org/hibernate/reactive/types/JoinColumnsTest.java

```diff
@@ -210,9 +209,8 @@ public void testTransientReferenceExceptionWithMutiny(TestContext context) {
 				.onItem().invoke( session -> context.fail( "Expected exception not thrown" ) )
 				.onFailure().recoverWithUni( throwable -> {
 					context.assertNotNull( throwable );
-					context.assertEquals( CompositeException.class, throwable.getClass() );
-					context.assertEquals( IllegalStateException.class, throwable.getCause().getClass() );
-					context.assertEquals( TransientPropertyValueException.class, throwable.getCause().getCause().getClass() );
+					context.assertEquals( IllegalStateException.class, throwable.getClass() );
```

Comment:
Mutiny 0.15.0 fixes composite exceptions so we only wrap when actually needed.

Reply:
Awesome!